src/frontend/komponenter/Oppgaver/FilterSkjema.tsx

```diff
@@ -0,0 +1,227 @@
+import React, { useState, useEffect } from 'react';
+import { Select } from 'nav-frontend-skjema';
+import { useOppgaver } from '../../context/OppgaverContext';
+import {
+    OppgavetypeFilter,
+    EnhetFilter,
+    SaksbehandlerFilter,
+    GjelderFilter,
+    PrioritetFilter,
+    IOppgave,
+} from '../../typer/oppgave';
+import { Knapp } from 'nav-frontend-knapper';
+import './visoppgave.less';
+import { ISaksbehandler } from '../../typer/saksbehandler';
+import { RessursStatus, Ressurs } from '../../typer/ressurs';
+import moment from 'moment';
+import Datovelger from '../Felleskomponenter/Datovelger/Datovelger';
+import { datoformatNorsk } from '../../utils/formatter';
+
+type IOppgaverFilter = {
+    name: string;
+    label: string;
+    values: string[];
+    selectedValue: string;
+};
+
+const initialFiltre = (innloggetSaksbehandler?: ISaksbehandler): IOppgaverFilter[] => {
+    return [
```

Comment:
Is this static or dynamic filters?
If static; maybe use an object with key as the name to avoid looping through a list - just make lookups directly:
```
{
   enhet: {
            name: 'Enhet',
            label: 'Enhet',
            values: Object.values(EnhetFilter).map(v => v.toString()),
            selectedValue: EnhetFilter.Alle,
        },
    oppgavetype: {
            name: 'Oppgavetype',
            label: 'Oppgavetype',
            values: Object.values(OppgavetypeFilter).map(v => v.toString()),
            selectedValue: OppgavetypeFilter.Alle,
        },
        behandlingstema: {
            name: 'Behandlingstema',
            label: 'Gjelder',
            values: Object.values(GjelderFilter).map(v => v.toString()),
            selectedValue: GjelderFilter.Alle,
        }...
}
```

Reply:
Good catch! I will make it an object👍